### PR TITLE
5 finish the application header

### LINKED ISSUE #5: Finish the application header
- [x] Modify the application header to match the mockups
- [x] If not done already, make the app name / logo a link to the home page
- [ ] Add the application logo if it has been created
- [x] Use the chosen colors if they have been chosen
- [x] Try to add the language selection button, if too hard, leave for a future and create a new issue
- [x] Don't do anything with the search bar at the moment (future issue)

### PR DESCRIPTION
**Pull request by Smartycrouton**
## Describe your changes
I have created a new header and the possibility of changing the application from english to french

## Screenshots (if appropriate):
Screenshot of the french then english front
Ordi - fr
Ordi - en

Screenshot of the french then english front for a mobile
Tel - fr
Tel - en

## Issue ticket number
Issue number 5
Closes #

## Checklist before requesting a review
- [X ] I have performed a self-review of my code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have written unit tests for my code if applicable
- [X ] I have increased the version number in the front/package.json and in the back/api.yaml files

I hope you'll like it

### CODE REVIEW

File: back/config/config.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# MuseumGo configuration file
+[ARANGODB]
+HOST = tomansion.fr
+PORT = 8529
+DATABASE = MuseumGo-dev
+USER = PoukouPoukou
+PASSWORD = cyMq3kerU5TfxBtVjgL6nCJP2RK9pESv
```
Comment (Tomansion) [inline back/config/config.ini]: ( ͡ʘ ͜ʖ ͡ʘ)  ︻デ═一

File: front/src/locales/en.json [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
-    "message": "hello i18n !!",
+    "message": "hello world!!",
     "hello": "Hello from component!",
     "setVariable": "Set Variable",
     "variableCorrect": "The variable is correct",
     "variableIncorrect": "The variable is incorrect"
+  },
+  "Header": {
+    "app-name": "MuseumGo",
+    "search-bar": "Researche",
```
Comment (Tomansion) [inline front/src/locales/en.json]: Search

**Review by Tomansion — CHANGES_REQUESTED** [review]

File: front/src/components/AppHeader.vue [hunk 3]
```diff
@@ -1,27 +1,63 @@
...
+      font-weight: bold;
+  }
+
+  .app-name {
+    font-size: 24px;
+  }
+
+  .search-bar {
+    margin-left: 20px;
+  }
```
Comment (Tomansion) [inline front/src/components/AppHeader.vue]: Search bar is on the right (not in the center)

**Review by Tomansion — CHANGES_REQUESTED** [review]

File: front/src/locales/en.json (hunk 2 above)
Comment (Smartycrouton) [reply front/src/locales/en.json]: Done

**Comment by Smartycrouton** [comment]
Correction of the search bar after the comments of Tom

- 'researche' is changed by 'search'
- search-bar is now on the right

Web capture_21-1-2024_22422_localhost
Web capture_21-1-2024_22411_localhost

File: front/src/components/AppHeader.vue (hunk 3 above)
Comment (Smartycrouton) [reply front/src/components/AppHeader.vue]: Done

**Review by Tomansion — APPROVED** [review]

**Pull request merged**